src/System.Security.Principal.Windows/src/PinvokeAnalyzerExceptionList.analyzerdata.netcoreapp

```diff
@@ -0,0 +1 @@
+advapi32.dll!LsaNtStatusToWinError
```

Comment:
Why does this need baselined?

Reply:
The .netcoreapp2.0 was part of the filename.

Note this is legal in UWP RS3 so we could arguably add it to our OneCore baseline depending on how far back we support OneCore. I have never updated that baseline. For now exclusion is fine.

Reply:
Yes. Since we are building for netcoreapp2.0 and netcoreapp I added the netcoreapp exclusion since the other one is explicit to netcoreapp2.0. I was thinking of also removing the netcoreapp part but that would add the exclusion for all the other frameworks.